amass/binaryedge.go

```diff
@@ -62,21 +62,18 @@ func (be *BinaryEdge) startRootDomains() {
 }
 
 func (be *BinaryEdge) executeQuery(domain string) {
-	var err error
-	var url, page string
-
-	key := be.Enum().Config.GetAPIKey(be.String())
-	if key == nil {
+	api := be.Enum().Config.GetAPIKey(be.String())
+	if api == nil {
```

Comment:
I can see forgetting to set an API key being a common error. We should have log messages when the API key isn't supplied.

Reply:
All the data sources that use API keys now log a failure to acquire the correct values